mollie-payments-for-woocommerce/includes/mollie/wc/plugin.php

```diff
@@ -467,6 +502,11 @@ public static function getPluginUrl ($path = '')
     	return M4W_PLUGIN_URL . $path;
     }
 
+    public static function getPluginPath($path = '')
```

Comment:
Looks like this should not be the responsibility of this class.

Reply:
@XedinUnknown Have a look at the class, the class contains also getPluginUrl, so for now that's the right place where add the method.

Reply:
Ok, since this is the plugin class, this method makes sense here.